app/styles/scenariobuilder.scss

```diff
@@ -33,6 +33,35 @@
     padding: 8px 16px; // Extra pixel top and bottom for border on buttons
    }
 
+  .toggle {
+    margin-top: $padding;
+  }
+
+  .multiplytable {
```

Comment:
Hmmm should do, but the goal is not to make it full-width. The component multiplytable is a Vue generated div with computed values. The btn which toggles this table should be padded top and bottom. That's why this scss is here.